docs/resources/aws_account.md

```diff
@@ -13,16 +13,41 @@ Manages an AWS account in CloudSecure.
 ## Example Usage
 
 ```terraform
-resource "illumio-cloudsecure_aws_account" "account_example" {
+/*
+  ⚠️ Internal Use Only: This resource is intended for internal module development
```

Comment:
Have you tried moving this warning note comment outside of the code block?
Same with the one below?
If that works, you could have more control over the formatting and make sure that the links work.

Same for the other resources.

Reply:
this is part of the example in `examples/resources/illumio-cloudsecure_aws_account.tf`
modifying markdown wont work as next time we run generate it will get overwritten. 

In order to achieve this kind of mix of markdown and terraform tf we have to migrate the docs to use templates.
https://github.com/hashicorp/terraform-plugin-docs?tab=readme-ov-file#usage

i tried doing it and its generating tmpls and also changing the structure and names of files and folders.
We can attempt that in a new pr. For now can we have it this way. and move it with the migration of docs.?

Reply:
another way is to use the description section of the resource to key in this information as it will be rendered as markdown.

Reply:
OK, let's keep it as is for now.